packages/@rescript/runtime/Stdlib_TypedArray.res

```diff
@@ -1,106 +1,508 @@
 @notUndefined
 type t<'a>
 
-@get_index external get: (t<'a>, int) => option<'a> = ""
-@set_index external set: (t<'a>, int, 'a) => unit = ""
+/**
+`get(typedArray, index)` returns the element at `index` wrapped in `Some`, or `None` when the position is out of bounds.
+
+See [`TypedArray.prototype.at`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/at) on MDN.
+
+## Examples
+
+```rescript
+let view = Int32Array.fromArray([1, 2, 3])
+TypedArray.get(view, 0) == Some(1)
+TypedArray.get(view, 10) == None
+```
+*/
+@get_index
+external get: (t<'a>, int) => option<'a> = ""
+
+/**
+`set(typedArray, index, value)` writes `value` at `index` in place.
+
+See [`TypedArray.prototype.set`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/set) on MDN.
```

Comment:
`Stdlib_TypedArray.set` does not correspond to [`TypedArray.prototype.set(array, targetOffset)`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/set) - the latter is for setting multiple values at some offset inside a typed array.

```suggestion
`set(typedArray, index, item)` sets the provided `item` at `index` of `typedArray`.

Beware this will *mutate* the array.
```